### PR TITLE
outgoing templated fields

### PR DESCRIPTION
**Pull request by chriku**

### CODE REVIEW

File: sync/src/main/kotlin/gropius/sync/AbstractSync.kt [hunk 1]
```diff
@@ -150,6 +150,18 @@ abstract class AbstractSync(
         imsProject: IMSProject, issueId: String, newState: IssueState, users: List<User>
     ): TimelineItemConversionInformation?
 
+    /**
+     * Incorporate a templated field change
+     * @param imsProject IMS project to sync
+     * @param issueId GitHub ID of the issue
```
Comment (nk-coding) [inline sync/src/main/kotlin/gropius/sync/AbstractSync.kt:156]: why is there a GitHub id in the AbstractSync?

File: sync/src/main/kotlin/gropius/sync/AbstractSync.kt [hunk 2]
```diff
@@ -844,6 +866,37 @@ abstract class AbstractSync(
...
+     * @param timeline Timeline of the issue
+     * @param imsProject IMS project to sync
+     * @param issueInfo Issue to sync
+     */
+    private suspend fun syncOutgoingTemplatedFields(
+        timeline: List<TimelineItem>, imsProject: IMSProject, issueInfo: IssueConversionInformation
+    ) {
+        val virtualIDs = mapOf<TimelineItem, String>()//For future features
+        val relevantTimeline = timeline.mapNotNull { it as? TemplatedFieldChangedEvent }
+        if (relevantTimeline.isEmpty()) return
```
Comment (nk-coding) [inline sync/src/main/kotlin/gropius/sync/AbstractSync.kt:880]: brackets pls

**Review by nk-coding — CHANGES_REQUESTED** [review]

File: sync/src/main/kotlin/gropius/sync/AbstractSync.kt (hunk 1 above)
Comment (chriku) [reply sync/src/main/kotlin/gropius/sync/AbstractSync.kt:156]: 62aa13dffbc73e4f6bd19814988eb3b515342a08

File: sync/src/main/kotlin/gropius/sync/AbstractSync.kt (hunk 2 above)
Comment (chriku) [reply sync/src/main/kotlin/gropius/sync/AbstractSync.kt:880]: 4372292e8abc99eb4ca7cd6c6d2f1c5ec6a3e894

**Pull request closed without merging**